plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -113,11 +113,17 @@ public class DefaultJdbcMetadata
 
     private final AtomicReference<Runnable> rollbackAction = new AtomicReference<>();
 
-    public DefaultJdbcMetadata(JdbcClient jdbcClient, boolean precalculateStatisticsForPushdown, Set<JdbcQueryEventListener> jdbcQueryEventListeners)
+    private final SyntheticColumnHandleBuilder builder;
+
+    public DefaultJdbcMetadata(JdbcClient jdbcClient,
+            boolean precalculateStatisticsForPushdown,
+            Set<JdbcQueryEventListener> jdbcQueryEventListeners,
+            SyntheticColumnHandleBuilder builder)
     {
         this.jdbcClient = requireNonNull(jdbcClient, "jdbcClient is null");
         this.precalculateStatisticsForPushdown = precalculateStatisticsForPushdown;
         this.jdbcQueryEventListeners = ImmutableSet.copyOf(requireNonNull(jdbcQueryEventListeners, "queryEventListeners is null"));
+        this.builder = requireNonNull(builder, "aliasFormatter is null");
```

Comment:
nit: the field name in the message should match the actual name. `builder` is very generic, `columnHandleBuilder` maybe would be better.
```suggestion
        this.builder = requireNonNull(builder, "builder is null");
```

Reply:
Thanks. Good spotting. Leftover from a refactor - intellij does not change strings automatically ...
Will also fix the other place.